package.json

```diff
@@ -110,17 +104,97 @@
     "tslint-plugin-prettier": "^2.0.1",
     "typescript": "~5.0.4"
   },
+  "sideEffects": ["style/*.css", "style/index.js"],
+  "styleModule": "style/index.js",
+  "publishConfig": {
+    "access": "public"
+  },
   "jupyterlab": {
-    "extension": "lib/index.js",
+    "discovery": {
+      "server": {
+        "managers": ["pip"],
+        "base": {
+          "name": "@jupyterlab/latex"
```

Comment:
This seems incorrect to me
The template has it as `{{python_name}}`
```
"discovery": {
            "server": {
                "managers": [
                    "pip"
                ],
                "base": {
                    "name": "{{ python_name }}"
                }
            }
        },
```
Is `@jupyterlab/latex` name of the Python package?

Reply:
You're right. It's been fixed.